agent/xds/rbac_test.go

```diff
@@ -824,6 +825,22 @@ func TestMakeRBACNetworkAndHTTPFilters(t *testing.T) {
 				),
 				testSourceIntention("*", structs.IntentionActionDeny),
 			),
+			v2TrafficPermissions: &pbproxystate.TrafficPermissions{
```

Comment:
It looks like there are a bunch of L7 tests in this file that don't have v2 equivalents. Did you consider adding v2 versions of these tests?

Reply:
Yes, this is a quick follow up I intend to do. The primary issue is that many of them include "deny", and given the tweaks to the deny/exclude paradigm, I need to do the correct translation using excludes.

Reply:
We should add the above-mentioned tests, excludes tests and additional tests for cases that weren't expressible in v1. Given the importance of traffic permissions, we should try to get as close as possible to 100% coverage.